### PR TITLE
P3 view audit log

### PR DESCRIPTION
**Pull request by diana-macrometa**
This PR adds information about the Audit Log in the GUI.

Reviewers - Please respond to my comments.

### CODE REVIEW

File: docs/photoniq/p3/view-p3-audit-log.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+sidebar_position: 60
+title: View P3 Audit Log
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+After you have created one or more policies, you can view the P3 audit log on the dashboard.
```
Comment (diana-macrometa) [inline docs/photoniq/p3/view-p3-audit-log.md]: What all is logged? We should tell the user what this log tracks.

Is it related to the Alerts at all? Like, will an alert move to the audit log once it is fixed?

File: docs/photoniq/p3/view-p3-audit-log.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+3. Scroll down to the Audit Log table to view the log.
+
+![View P3 Audit Log](/img/photoniq/p3/p3-view-audit-log.png)
+
+## Audit Log Fields
+
+The following fields list information about the audit log lines:
+
+- **URL** - The URL the log line applies to.
+- **Status** - The status at the time of logging. Possible statuses are Success, Failed, and XXX.
```
Comment (diana-macrometa) [inline docs/photoniq/p3/view-p3-audit-log.md]: What statuses are possible and what does each one mean?

File: docs/photoniq/p3/view-p3-audit-log.md (hunk 1 above)
Comment (dwivedianurag) [reply docs/photoniq/p3/view-p3-audit-log.md]: It is not related to alert . This audit logs the requests and their path and the queries for which we have applied the optimizations and whern

File: docs/photoniq/p3/view-p3-audit-log.md (hunk 2 above)
Comment (dwivedianurag) [reply docs/photoniq/p3/view-p3-audit-log.md]: status can be Success and Failed -- Success means successfully applied, failed means optimizations are not applied  on the configured pages

**Review by dwivedianurag — APPROVED** [review]
ok

**Pull request merged**